devito/finite_differences/tools.py

```diff
@@ -159,51 +157,46 @@ def symbolic_weights(function, deriv_order, indices, dim):
             for j in range(0, len(indices))]
 
 
-def generate_indices(func, dim, diff, order, stagger=None, side=None):
-
+def generate_indices(func, dim, order, side=None, x0=None):
     # If staggered finited difference
```

Comment:
expand like, if function...

Reply:
?

Reply:
that's nitpicking. I think (I think) he's saying that throughout the code base we have either `function` or `f`, not `func`